class/authenticator.ts

```diff
@@ -42,15 +42,24 @@ export class Authenticator implements IAuthenticator {
     return authenticator;
   }
 
-  async init(entropy: string) {
-    const buffer = Buffer.from(entropy, ENCODING);
+  async init({ entropy, mnemonic }: { entropy?: string; mnemonic?: string }) {
+    if (entropy === undefined && mnemonic === undefined) {
+      throw new Error('Not provided entropy or mnemonic');
+    }
+    const _entropy = mnemonic === undefined ? entropy : mnemonicToEntropy(mnemonic).toString(ENCODING);
+
+    if (_entropy === undefined) {
+      throw new Error('Couldn`t get entropy');
```

Comment:
if those errors are visible for users we should keep them in locale files

Reply:
there are not visible for user